### PR TITLE
Fix for issue #45, added functionality to configure source of timestamps

### PR DESCRIPTION
**Pull request by katarzyna-z**
Fix for issue #45 , added functionality to configure source of timestamps.

### CODE REVIEW

**Comment by PatrykMatyjasek** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -64,8 +64,94 @@ This builds the plugin in `/build/rootfs/`
 ## Documentation
 << @TODO
 
+###Configuration fields in task manifest for InfluxDB plugin
+* **host** - InfluxDB host
+* **port** - InfluxDB port
+* **user** - InfluxDB user
+* **password** - InfluxDB password
+* **publish_timestamp** - optional parameter to define if timestamps which were assigned for metrics are published  (parameter set on true - default option) or timestamps created by publisher plugin are used (parameter set on false).
```
Comment (andrzej-k) [inline README.md]: Consider rewording - maybe to something like this: "optional parameter to define if plugin shall publish metric collection timestamp (default - parameter set to true) or not (parameter set to false). In the latter case timestamp will be set by InfluxDB."

**Comment by andrzej-k** [comment]
Please add or update unit tests to cover this new config param.

File: influx/influx.go [hunk 2]
```diff
@@ -167,9 +169,16 @@ func (f *influxPublisher) Publish(contentType string, content []byte, config map
 			}
 		}
 
+		//check if timestamp source is defined in plugin configuration and get timestamps which were assigned for metrics if needed
+		var timestamp time.Time
+		timestampSource, timestampSourceExist := config[publishTimestamp]
```
Comment (andrzej-k) [inline influx/influx.go]: Consider changing "timestampSourceExist" to "ok"

Comment (katarzyna-z) [inline influx/influx.go]: Changed

**Comment by katarzyna-z** [comment]
Added unit and integration tests to cover new config param.

File: README.md (hunk 1 above)
Comment (katarzyna-z) [inline README.md]: Changed

**Comment by geauxvirtual** [comment]
This PR will need to be updated with the changes included in PR #44 that updated the client to v2 for InfluxDB.

**Comment by katarzyna-z** [comment]
This PR has been updated.

**Comment by marcin-krolik** [comment]
This PR is no longer needed since issue with publishing timestamp in correct way has been resolved. Suggest closing it.

**Comment by katarzyna-z** [comment]
@marcin-krolik I agree with you.

**Pull request closed without merging**